src/GeoVector.h

```diff
@@ -28,37 +31,47 @@ class GeoVector {
 
   inline GeoVector<T> operator/(const GeoVector<T>& other) const;
 
-  inline GeoVector<T> operator+(T other) const;
+  inline GeoVector<T> operator+(const T other) const;
```

Comment:
Do these need to be const? Do we ever expect T to be a byref or pointer type?

Reply:
I uh...might have missed that. Will rectify.